### DISCUSSION TITLE
Providers connections not showing up despite being installed and airflow restarted

### DISCUSSION THREAD

**Discussion by bab014** [apache/airflow#22286] [Q&A]
Hello there,

I've installed both the SFTP provider package as well as the SSH one and then restarted my airflow scheduler & webserver.

After restarting and looking to create the specific connections , they ar3e not showing up.
![image](https://user-images.githubusercontent.com/61920876/158392922-2ffc754f-402f-48cb-8481-50f81707de31.png)

I've also checked that they show up in the `providers` section in the `Admin` pane and they are in fact there.
![image](https://user-images.githubusercontent.com/61920876/158393173-3162b280-39c5-4d70-a48f-e077bf4c0001.png)

I've restarted the server multiple times now but nothing seems to be working. Any thoughts/ideas/help would be greatly appreciated.

**Comment by potiuk** [comment]
set debug mode on Airflow (see configuration docs on how to do it) and see if there are no errors with importing Providers. In recent versions of Airlfow they will appear without changing log level so you might want to also try to upgrade (You did not explauin which verison of Airlfow you use and what deployment you use either).

**Reply by bab014** [reply]
I'm sorry about that. 
It's a local install of airflow 2.2.4

**Reply by potiuk** [reply]
Error at import is most probably the reason - you should be able to find some warnings in the logs (or when you add debug logs you should see it here).
